src/platform/packages/shared/kbn-esql-ast/src/types.ts

```diff
@@ -372,6 +373,17 @@ export interface ESQLSource extends ESQLAstBaseItem {
   selector?: ESQLStringLiteral | undefined;
 }
 
+/**
+ * Represents any expression wrapped in parentheses.
+ *
+ * ```
+ * FROM ( <query> )
```

Comment:
```suggestion
 * ```
 * FROM ( <query> )
 * ```
```